vraptor-site/content/en/docs/one-minute-guide.html

```diff
@@ -158,7 +158,7 @@
 
 ##Questions
 
-While studying VRaptor, should you have any doubts please refer to <a href="http://questions.vraptor.org/">questions.vraptor.org</a>, or visit our discussion list at <a href="http://groups.google.com/group/caelum-vraptor-en">discussion list</a>.
+While studying VRaptor, should you have any doubts please visit our discussion list at <a href="http://groups.google.com/group/caelum-vraptor-en">discussion list</a>.
```

Comment:
> please visit our discussion list at *discussion list*

could you please remove the ambiguity? And reply the same in the other statements 

ps: know it's already there before your fix, it's just a *Boy Scouts role* matter (: